### DISCUSSION TITLE
Ruby lesson- Project: Event Manager (Specify project location to retain consistency with other projects)

### DISCUSSION THREAD

**Discussion by Maefire** [TheOdinProject/curriculum#25152] [Ideas]
For every other project from Foundations up until now, it was assumed that you would create a repository on GitHub, then clone that repo, and commit/push from the cloned folder. In this, we are simply instructed to "create it wherever we want". I would suggest it is at least encouraged to create the project folder in the aforementioned "repo" directory. 

Perhaps even create the repo with the "event_manager" file name, then clone it into the repo directory, before beginning the tutorial/project assignment. Again, this is mainly for consistency. With the current wording, I ended up having to figure out on my own how to push these files onto GitHub, from a completely new directory, when I could have saved myself the trouble by cloning and going from there.

**Comment by CouchofTomato** [comment]
Hey @Maefire 

Happy to receive a PR on this in the relevant lesson.

**Comment by KevinMulhern** [comment]
Looks like this has been resolved, thanks @Maefire 💪
